cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -688,6 +688,7 @@ ThunkAction<Promise<void>, {}, {}, AnyAction> {
             });
 
             const data = await job.frames.get(toFrame, fillBuffer, frameStep);
+            if (data === undefined) return;
```

Comment:
``job.frames.get()`` is not expected to return ``undefined`` at all. @azhavoro right?
Probably there is a problem exact in this method

Reply:
Yes, frames.get() should always return Instance of FrameData.